upgrade/scripts/sls/expert_mode.md

```diff
@@ -35,9 +35,9 @@ The upgrader requires only two inputs:
 An example of this would be:
 
 ```bash
-./sls_updater_csm_1.2.py \
-    --sls-input-file sls_input_file.json \
-    --bican-user-network-name CAN
+ncn# ./sls_updater_csm_1.2.py \
```

Comment:
The command line prompts in this document don't match https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts-inside-of-directions

Reply:
This was marked resolved, but was not fixed. I am correcting it as part of this PR:
https://github.com/Cray-HPE/docs-csm/pull/2082